docs/commands.md

```diff
@@ -0,0 +1,13 @@
+# Command Arguments
+
+## Banphrases
+
+`--length`/`--time`/`--duration` - sets the timeout length of for the phrase (in seconds). Default = 600 seconds\
```

Comment:
I would prefer if you could either use a bullet point list or use the "double space at end of line" instead of the backslash to create the linebreaks.

Reply:
Using a bullet point list wouldn't be consistent with the variables doc. I've used the double spaces.